kinetic/distribution.yaml

```diff
@@ -507,6 +507,21 @@ repositories:
       url: https://github.com/po1/axcli-release.git
       version: 0.1.0-0
     status: maintained
+  axis_camera:
+    doc:
+      type: git
+      url: https://github.com/ros-drivers/axis_camera.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/clearpath-gbp/axis_camera-release.git
```

Comment:
Should this be hosted on ros-gbp unstead ? (as an orphaned package, my guess is that it should be in an organization where we can give access to orphaned package maintainers)

Reply:
sorry fixed.